selectel/resource_selectel_domains_domain_v1.go

```diff
@@ -0,0 +1,106 @@
+package selectel
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	v1 "github.com/selectel/domains-go/pkg/v1"
+	"github.com/selectel/domains-go/pkg/v1/domain"
+)
+
+const domainsEndpoint = "https://api.selectel.ru/domains/v1"
```

Comment:
This constant will be used in other `domains` resources so it's better to move it somewhere else.
Since Domains API only has a single URL like Resell API can we move it into the `domains-go` library and do something like https://github.com/selectel/go-selvpcclient/blob/master/selvpcclient/resell/v2/client.go#L17